src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -289,6 +289,8 @@ export class FindAndReplacePlugin extends UIPlugin {
         sheetIdFrom: this.getters.getActiveSheetId(),
         sheetIdTo: selectedMatch.sheetId,
       });
+      // We do not want to reset the selection at finalize in this case
+      this.isSearchDirty = false;
```

Comment:
I think that the index will not be updated if the side panel is open and the use decides to change sheet by themselves.

Reply:
Yeah taht's GSheet behaviour. The search index doesn't change when we switch sheet manually

Reply:
I prefer to see the index change with the change of sheet tbh but I don't mind. @Francois-frgi Could you give us your thoughts on this?

If I start the find and replace panel and look for a term - I decide to change sheets manually; Do I want:
- the search index to be updated so that it focuses the first match of the new sheet I opened?
- or I do not touch the matched index and if I press next/previous value, it could change the sheet automatically?